tests/Type/WebMozartAssert/data/data.php

```diff
@@ -117,11 +117,19 @@ public function doFoo($a, $b, array $c, iterable $d, $e, $f, $g, $h, $i, $j, $k,
 
 		Assert::integerish($ad);
 		$ad;
-	}
+
+        Assert::implementsInterface($q, Baz::class);
```

Comment:
Hi, you should test variable called `$ae` here and add it to the parameter list of `doFoo` method. Nothing else needs to change, the implementation is fine, but this is inconsistent with other tested methods.

Reply:
I will change that. Could you probably explain what the idea is behind that method `doFoo` as i definitely dont get why phpstan does not argue about the method call without passing parameters to it for example in here:
https://github.com/phpstan/phpstan-webmozart-assert/blob/13fe743c82c6321d51870f513c7c71cf432f4ffe/tests/Type/WebMozartAssert/data/data.php#L63

Reply:
In this case it's even calling an undefined function. It doesn't complain because this is actually a unit test of a single rule. These errors are detected by a different rule. This is the way this extension is testing its implementation.